sites/main-site/src/content/blog/2024/maintainers-minutes-2024-07-26.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Maintainers Minutes: August 2024'
+subtitle: 'Keeping you informed of the latest maintainers discussions'
+pubDate: 2024-08-14T12:00:00+02:00
+headerImage: '/assets/images/blog/maintainers-minutes-2024-07-26/maintainers_duck.png'
+headerImageAlt: "Cartoon yellow rubber duck with nf-core logo badge on it's body with the nf-core logo."
+embedHeaderImage: true
+authors:
+    - 'jfy133'
+    - 'FriederikeHanssen'
+label:
+    - 'maintainers'
+---
+
+The 'Maintainers Minutes' aims to further give insight into the workings of the [nf-core maintainers team](/governance#maintainers)
+by providing brief summaries of the monthly team meetings.
+
+## Review of action points from last time
+
+We got about half way through our action points from last time.
+
+Slack channels were created for the new working groups: - #wg-meta-pipelines - #wg-modules-update - #wg-test-data-task-force
+
+Please join if you want to be involved in one of those.
+
+## More (or less) testing
+
+Júlia encountered several failing tests on nf-core/crisprseq with conda and singularity. This was caused by tests only running
+on docker. Moving forward we agreed that at least some tests should also be run
+with `singularity` and `conda`.
+
+We discussed several ideas. On the one hand we'd like to make sure everything works at all times, on the other hand running tons of tests
+makes development slow, costs money, and is not great for the planet 🌳. As a compromise, we agreed on the following: - Singularity & conda tests are run on PR to master - Make singularity required - Conda optional opt-out - Add `workflow dispatch` button for easier manual testing
+
+## Nf-test best practices
+
+Nicolas brought up that snapshots should be readable to quickly determine that they do what they should. Since the order of snapshots in the `.snap` file
+does not correspond to the order they are defined in the test file, having one snapshot per file in the same test
+scrambles the entries all over the place.
+
+Nicolas will make updates to the guidelines that all assertions should go into a single snapshot per test.
+
+## Bulk updates of modules
+
+Júlia is coordinating the bulk update of modules in https://github.com/nf-core/modules/issues/5828 (join #wg-modules-update to stay tuned)
+
+This includes: - meta.yml accounting for the channel structure (https://github.com/nf-core/modules/pull/5867) - subworkflows - bio.tools identifiers - edam ontologies for all input/output files - removing defaults from conda channel - adding lock files - adding version topic channels - adding lint for stub tests for all modules
```

Comment:
```suggestion
This will include a lot of new exciting things:

  - improved `meta.yml` layout accounting for the channel structure (https://github.com/nf-core/modules/pull/5867) 
  - subworkflows 
  - bio.tools identifiers (for better linked metadata across registries)
  - edam ontologies for all input/output files  (for better standardisation of file types)
  - removing defaults from conda channel (to reduce chances of ToS violations)
  - adding lock files (for better conda dependency version stability)
  - adding version topic channels (for simplified version reporting in pipelines)
  - adding lint for stub tests for all modules (to help us move to having complete 'dry run' functionality in pipelines)
```